actionpack/CHANGELOG.md

```diff
@@ -184,7 +184,7 @@
 
 *   Matches behavior of `Hash#each` in `ActionController::Parameters#each`.
 
-    Rails 5.0 introduced a bug when looping through controller params using an each block. The value would be returned as a string e.g.
+    Rails 5.0 introduced a bug when looping through controller params using each block. The value would be returned as a string e.g.
```

Comment:
I think this might need some rewrite, and is a bit unrelated to the change you proposed.

What do you think of:

```suggestion
    Rails 5.0 introduced a bug when looping through controller params using `each`. Only the key of params hash get passed to the block, e.g.
```

Reply:
I am not completely aware of this change but as per the example mentioned in the changelog, changes you suggested looks good to me.

I am updating that here. Also, just updating `key` to `keys` in the above change suggested.

Reply:
@sikachu : Fixed.